src/js/components/aboutTheDataSidebar/AboutTheDataDrilldown.jsx

```diff
@@ -20,13 +20,28 @@ const propTypes = {
 const AboutTheDataDrilldown = ({
     section, name, clearDrilldown, slug
 }) => {
+    // figure out if there is a param
+    const stripUrl = () => {
+        const url = new URL(window.location.href);
+        if (url.search !== '') {
+            const test1 = window.location.href.includes("about-the-data=");
+            const test2 = window.location.href.includes("glossary=");
```

Comment:
@nick-torres Are test1 and test2 necessary?  Are there cases where we wouldn't have a "?" if "about-the-data" or "glossary" is in the url? Test3 should cover the test1 and test2 cases.

Reply:
yes because you can have the agency page with would be ?fy=2020&about-the-data=file-a, then what if you open a glossary page for sharing from that

Reply:
Oh yes, i'm saying that cases like "?fy=2020&about-the-data=file-a" still include the "?"

Reply:
yeah already changed it